test/debts.js

```diff
@@ -17,43 +18,85 @@ contract('Debts', function(accounts) {
     const value = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
     return Promise.resolve()
-    .then(() => debts.borrow(value, {from: borrower}))
-    .then(() => asserts.throws(debts.borrow(1, {from: borrower})));
+    .then(() => debts.borrow(value, {from: DEFAULT_BORROWER}))
+    .then(() => asserts.throws(debts.borrow(1, {from: DEFAULT_BORROWER})));
   });
 
   it('should emit Borrowed event on borrow', () => {
-    const borrower = accounts[3];
     const value = 1000;
     return Promise.resolve()
-    .then(() => debts.borrow(value, {from: borrower}))
+    .then(() => debts.borrow(value, {from: DEFAULT_BORROWER}))
     .then(result => {
       assert.equal(result.logs.length, 1);
       assert.equal(result.logs[0].event, 'Borrowed');
-      assert.equal(result.logs[0].args.by, borrower);
+      assert.equal(result.logs[0].args.by, DEFAULT_BORROWER);
       assert.equal(result.logs[0].args.value.valueOf(), value);
     });
   });
 
-  it('should allow to borrow');
+  it('should allow to borrow', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: DEFAULT_BORROWER}))
+    .then(() => debts.debts(DEFAULT_BORROWER))
+    .then(asserts.equal(value));
+  });
+
+  it('should emit Repayed event on repay', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: DEFAULT_BORROWER}))
+    .then(() => debts.repay(DEFAULT_BORROWER, value, {from: OWNER}))
+    .then(result => {
+      assert.equal(result.logs.length, 1);
+      assert.equal(result.logs[0].event, 'Repayed');
+      assert.equal(result.logs[0].args.by, DEFAULT_BORROWER);
+      assert.equal(result.logs[0].args.value.valueOf(), value);
+    });
+  });
 
-  it('should emit Repayed event on repay');
+  it('should not allow owner to borrow', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: OWNER}))
+    .then(() => debts.debts(OWNER))
+    .then(asserts.equal(0));
+  });
 
-  it('should not allow owner to borrow');
+  it('add: should not allow not owner to repay', () => {
+    const borrower = accounts[4];
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: DEFAULT_BORROWER}))
+    .then(() => debts.repay(DEFAULT_BORROWER, value, {from: DEFAULT_BORROWER}))
+    .then(() => debts.repay(DEFAULT_BORROWER, value, {from: borrower}))
+    .then(() => debts.debts(DEFAULT_BORROWER))
+    .then(asserts.equal(value));
+  });
 
-  it('should not allow not owner to repay');
+  it('add: should fail on overspend when repaying', () => {
+    const value = 1;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: DEFAULT_BORROWER}))
+    .then(() => asserts.throws(debts.repay(DEFAULT_BORROWER, value + 1, {from: OWNER})));
+  });
 
-  it('should direct you for inventing more tests');
+  it('add: should (not) allow to borrow 0', () => {
```

Comment:
If there was a test of 'should allow to borrow 0' how it would be different from this one?